lib/geo/GeoStyle.js

```diff
@@ -189,24 +192,26 @@ normalizeAndSeparateLayers(STYLE_BASE_AERIAL);
 
 // MOVE SOURCES
 
-function addTippecanoeSource(style, id, minLevel, maxLevel, crossfade = 0) {
+function addTippecanoeSource(style, id, minLevel, maxLevel, crossfade = 0, options = {}) {
```

Comment:
this isn't necessary yet, strictly speaking, but it will help if we want to introduce clustering for collisions in the near future (which we well might, to help reduce confusion here...)